robert.json

```diff
@@ -1 +1 @@
-"Name": "Robert Guillen"
+name = "Robert Guillen"
```

Comment:
JSON files are more or less a key value pairs within curly bracelets.
A valid JSON would look something like this:
```
{
    "name": "somename",
    "another thing": "thing two",
    "somenumber": 10
}
```

Notice the commas, also the `key` parts (the left hand side of colon) are quoted (`"something"`), and the `value` parts (the right hand side) are either quoted or could be some other types: such as integers, lists or could be even another key-value pairs!

- [ ] Try to have this file under `data` folder
- [ ] Also make this json file be a valid one